mautic/mautic
# TPROD-319 - LeadBundle - Import

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Check that everything relating to importing works - contacts and companies, with all the field matching, segment assignment etc. Check browser and cron based imports.



- Import contacts list is rendered correctly (with and without import history)
- Import contacts list can be sorted
- Import details page is rendered correctly
- The import form is rendered correctly
- Importing the file in the browser will display the same way as it does on the 5.x branch
- Trying to import without adding a csv file will display an error
